FCCEUD-50 update instructions for creating a new Product (#586)
redhataccess/pantheon: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_creating-a-product.adoc

= Creating a product

[role="_abstract"]
Before you can assign a product name to the metadata of a module or assembly, the product must exist in {ProductName}.

> To create a new product, do the following:

Reviewer: ```suggestion
```

.Procedure

. Navigate to menu:Products[New Product].

> . Enter the following information, items marked with a star (*) are mandatory:

Reviewer: star -> asterisk

[horizontal]
  Product Name *:: The text that appears in {ProductShortName}, the document preview, and in user-facing content. This name is replaced by the value of the `:pantheonproduct:` attribute in user-facing content that {ProductShortName} generates.
  Product URL Fragment *:: This string appears in the user-facing URL for any content that belongs to this product and version. A URL fragment can include letters, numbers, hyphens, periods and underscores.
  Product Description:: A description of the product.

>   Product Version *:: The product version. This value is replaced by the value of the `:pantheonversion:` attribute in user-facing content that {ProductShortName} generates.

Reviewer: > This value is replaced by the value of the `:pantheonversion:` attribute in user-facing content that {ProductShortName} generates.

This is wrong. Product version refers to the version of the Product, not Pantheon's version number. E.g. Product name = "RHEL" ; Product Version = "8.4". Or like the example farther down: `Product Version:: 4.4-Beta`

Author: ok, since I don't see any attributes list in our Pantheon repo, I'm simply using generic terms product version or version number in this description

.Procedure

. Navigate to menu:Products[New Product].
. Enter the following information, items marked with a star are mandatory:

>   Product Name *:: The text that appears in {ProductShortName}, the document preview, and in user-facing content.

Reviewer 2: I think this would be a good place to include a blurb like the following: "The product name is also injected into any associated document built by Pantheon via the :pantheonproduct: variable that is free for the author to use if they so choose."

That's probably pretty wordy but hopefully it gets the idea across; I'm sure you could come up with a better way to phrase it.

  Product URL Fragment *:: This string appears in the user-facing URL for any content that belongs to this product and version. A URL fragment can include letters, numbers, hyphens, periods and underscores.
  Product Description:: A description of the product.

>   Product Version *:: The product version.

Reviewer 2: Same as above, maybe indicate that the :pantheonversion: variable is injected into the documents that contains this value.

Author: adding to the description - "This value will be substituted wherever the `:pantheonversion:` attribute is used in user-facing content."

>   Version URL Fragment *:: This string appears in the user-facing URL for any content that belongs to this product and version. A URL fragment can include letters, numbers, hyphens, periods and underscores.

Reviewer: I'm OK with the asterisk convention if it's used everywhere else too. Otherwise I'd suggest one of the following:
* Use the same convention used elsewhere
* Create a Jira item to make changes to existing topics to use this convention, listing the scope of changes necessary.

. Click btn:[Save].

.Example

> * Product Name: .NET Core

Reviewer: Use parallelism wherever possible. The definition list above makes sense here too.

* Product URL Fragment: net_core
* Product Description: Documentation for the .NET Core framework.
* Product Version: 4.4-Beta
* Version URL Fragment: 4.4-beta

> Entering the above information creates a {ProductShortName} product named _.NET Core_. The URL for documentation that is associated with this product includes the URL Fragment _net_core_ to identify the product, and the Version URL Fragment is _4.4-beta_. The resulting URL for that module is:

Reviewer: Italics is usually a convention for user-replaced values in technical docs. I'd go back to either the monospace formatting or bold, but don't use italics.

. Navigate to menu:Products[New Product].
. Enter the following information, items marked with a star are mandatory:

>   Product Name *:: The text that appears in {ProductShortName}, the document preview, and in user-facing content. This name will be substituted wherever the `:pantheonproduct:` attribute is used in user-facing content that is generated by {ProductShortName}. 

Reviewer: ```suggestion
  Product Name *:: The text that appears in {ProductShortName}, the document preview, and in user-facing content. This name is replaced by the value of the `:pantheonproduct:` attribute in user-facing content that {ProductShortName} generates. 
```

  Product URL Fragment *:: This string appears in the user-facing URL for any content that belongs to this product and version. A URL fragment can include letters, numbers, hyphens, periods and underscores.
  Product Description:: A description of the product.

>   Product Version *:: The product version. This value will be substituted wherever the `:pantheonversion:` attribute is used in user-facing content that is generated by {ProductShortName}.

Reviewer: Same comment as above.
